io/io/src/TBufferMerger.cxx

```diff
@@ -69,7 +68,7 @@ void TBufferMerger::WriteOutputFile()
       merger.OutputFile(fName.c_str(), fOption.c_str(), fCompress);
    }
 
-   while (!done) {
+   for (;;) {
```

Comment:
http://stackoverflow.com/questions/5258081/whats-the-point-of-issuing-a-compiler-warning-for-whiletrue-and-not-issuing